source/common/http/http3/conn_pool.h

```diff
@@ -78,6 +84,11 @@ class ActiveClient : public MultiplexedActiveClientBase {
     }
   }
 
+  bool hasCreatedStream() const { return stream_attached_; }
```

Comment:
nit: I'd suggest renaming stream_attached_ to has_created_stream_ to match (or perhaps renaming the method to streamAttached()).